packages/plugin-renderer-puppeteer/src/plugins/server.js

```diff
@@ -15,8 +15,6 @@ class PuppeteerServer {
       (await getDevServer(this.compilation)).listen(offsetPort, async () => {
         console.info(`Started puppeteer prerender server at http://localhost:${offsetPort}`);
       });
-    } else {
```

Comment:
are we sure this should / needs to be deleted?  I would test with `yarn develop` but I think it's OK to leave this one out of this PR since it's a little out of scope, IMO.